Sources/Datadog/Tracing/AutoInstrumentation/TracingHeaderType.swift

```diff
@@ -8,11 +8,13 @@ import Foundation
 
 /// The type of the tracing header injected to requests.
 ///
-/// - `dd` - [Datadog's `x-datadog-*` header](https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces/?tab=browserrum#how-are-rum-resources-linked-to-traces).
-/// - `b3s` - Open Telemetry B3 [Single header](https://github.com/openzipkin/b3-propagation#single-headers).
-/// - `b3m` - Open Telemetry B3 [Multiple headers](https://github.com/openzipkin/b3-propagation#multiple-headers).
-public enum TracingHeaderType {
-    case dd
-    case b3s
-    case b3m
+/// - `datadog` - [Datadog's `x-datadog-*` header](https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces/?tab=browserrum#how-are-rum-resources-linked-to-traces).
+/// - `b3` - Open Telemetry B3 [Single header](https://github.com/openzipkin/b3-propagation#single-headers).
+/// - `b3multiple` - Open Telemetry B3 [Multiple headers](https://github.com/openzipkin/b3-propagation#multiple-headers).
+/// - `w3c` - W3C [Trace Context header](https://www.w3.org/TR/trace-context/#tracestate-header)
+public enum TracingHeaderType: Hashable {
+    case datadog
+    case b3
+    case b3multiple
+    case tracecontext
```

Comment:
documentation and enum name are mismatched here. Should this be `tracecontext` or `w3c`?